examples/docs_snippets/docs_snippets/legacy/dagster_pandas_guide/core_trip.py

```diff
@@ -1,3 +1,5 @@
+# pyright uses pandas 2 but docs_snippets is stuck on pandas 1 due to pins in dagster-duckdb-pandas
```

Comment:
this is a side effect of the pyright pins updating, not directly related other than that i needed to bump the pins to add the syrupy dep and remove the snapshottest dep

Reply:
thoughts on this @smackesey ?

Reply:
I see, I thought we'd successfully moved to pandas 2 everywhere. @jamiedemaria I think I remember talking about this transition with you?

In any case putting an ignore here is no biggie, but we should take care of the dagster-duckdb-pandas transition ASAP so we can get everything on pandas 2. Then this ignore will get auto-flagged as unnecessary and can be removed.

Reply:
https://github.com/dagster-io/dagster/pull/14172 this is what i found

Reply:
OK, here is a PR that updates the whole repo to support pandas 2 (and updates pins): https://github.com/dagster-io/dagster/pull/14956

@gibsondan This will merge before the current PR so you should regen the pyright pins on rebase and shouldn't need to have any special handling for pandas.